src/commands/InitCommand.ts

```diff
@@ -96,9 +103,9 @@ export class InitCommand implements yargs.CommandModule {
     // Protected Static Methods
     // -------------------------------------------------------------------------
 
-    protected static executeCommand(command: string) {
+    protected static executeCommand(command: string, cwd: string) {
```

Comment:
Previously, `npm install` command failed when a new folder is created for the new project